### PR TITLE
Add send results function

### PR DESCRIPTION
**Pull request by lucyrose93**
Added a function that sends the results of the survey to the clinician's email address

### CODE REVIEW

File: public/send-results.js
Comment (des-des) [inline public/send-results.js]: Hmm, can you be a bit more consistent in the way using session storage.

Ie https://github.com/lucyrose93/MakaTalk/blob/master/public/confirm-survey.js#L4

(I think using setItem is better)

Comment (des-des) [reply public/send-results.js]: Also, get is a verb. When you start a variable name with get, it makes it seem like a function

File: public/send-results.js
Comment (des-des) [inline public/send-results.js]: Hmm. Can you do this? I can not find this documented anywhere, and trying it in a repl fails..

**Review by des-des — CHANGES_REQUESTED** [review]

File: public/send-results.js
Comment (lucyrose93) [reply public/send-results.js]: Hi @des-des setItem was used in session-storage.js to store the type of survey

I am using getItem in send-results.js to retrieve the stored value, once the survey has been completed and the results need to be sent to the clinician 

Does that make sense? 

Will change the var name :)

File: public/send-results.js
Comment (lucyrose93) [reply public/send-results.js]: window.sessionStorage allows me to access the entire sessionStorage object. Otherwise I would have to specify all of the key value pairs that I want, for all the answers for both surveys. 
The syntax is the same for localStorage, as far as I know: https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage
An alternative is to have 2 conditions, with arrays for each set of questions, then you'd loop through and do getItem on each one. I did think about this. 
Let me know what you think 
----> Also something has broken so please don't merge  yet :p

**Comment by lucyrose93** [comment]
@des-des I've left some comments above about sessionStorage to explain my choices.

Please merge this if you don't see any problems with it. Thanks!

File: public/send-results.js
Comment (des-des) [reply public/send-results.js]:
```js
var storage = window.sessionStorage
var clinicianEmail = storage.getItem('clinician-email')
```

I think using `getItem` function is the suggested way of using session storage.

File: public/send-results.js
Comment (des-des) [reply public/send-results.js]: You could do this.
```js
var storage = window.sessionStorage
var allSessionData = {}
for (var i = 0; i < storage.length; i++) {
  var key = storage.key(i)
  allSessionData[key] = storage.getItem(key)
}
```
But the problem with both that approach and your current approach is that if you want to store other data in you session, its going to pop out unexpectedly.

Right now, the regex you are doing to create the email body is evil. You want to format your json in a nicer way? I expect that escaped characters in the json could do really weird things in the results.

One thing you could do is have a some functions

```js
var saveAnswer = (name, value) => {
  answers = json.parse(sessionStorage.getItem('answers'))
  answers[name] = value
  sessionStorage.setItem('answers', JSON.stringify(answers))
}
```

There are other things you can do also, like maintain an array of keys that you are interested in (I think this is what you have suggested)

**Comment by des-des** [comment]
@lucyrose93 I have responded to comments. I not really convinced by what you are doing.. If my comments don't make sense, or you do not see a way forward, I can give more detail

**Comment by lucyrose93** [comment]
Have opened a new PR with the recommended changes: https://github.com/lucyrose93/MakaTalk/pull/106

**Pull request closed without merging**
